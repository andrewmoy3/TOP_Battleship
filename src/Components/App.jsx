import { useState } from 'react'
import '../css/App.css'
import Banner from './Banner'
import Battlefield from './Battlefield'
import Game from './Game'
import Gameboard from './Gameboard'
import Guessesboard from './Guessesboard'

function App(props) {

  const { playerBoard, computerBoard, handleClick } = Game();
  return (
    <>
      <Banner />
      <div className='battlefields'>
        <div data-testid='playerBoard' id='playerBoard'>
          <Battlefield Gameboard={playerBoard} handleClick={handleClick} player='player'/>
          <Guessesboard Gameboard={playerBoard} handleClick={handleClick} player='player'/>
        </div>
        <div data-testid='computerBoard' id='computerBoard'>
          <Battlefield Gameboard={computerBoard} handleClick={handleClick} player='computer'/>
          <Guessesboard Gameboard={playerBoard} handleClick={handleClick} player='player'/>
        </div>
      </div>
    </>
  )
}

export default App
