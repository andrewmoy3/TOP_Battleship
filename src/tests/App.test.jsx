/* eslint-disable no-undef */
import { render, screen } from '@testing-library/react'
import App from '../Components/App'

describe('App tests', () => {
    it('tests for 2 10 by 10 grids', () => {
        const { getByTestId } = render(<App />)
        const playerBoard = getByTestId('playerBoard')
        const computerBoard = getByTestId('computerBoard')
        expect(playerBoard).toBeInTheDocument()
        expect(computerBoard).toBeInTheDocument()
    })

    it('App snapshot', () => {
        const { container } = render(<App />)
        expect(container).toMatchSnapshot() // why doesnt screen work?
    })
})
