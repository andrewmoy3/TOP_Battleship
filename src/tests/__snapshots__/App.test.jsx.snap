// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App tests > App snapshot 1`] = `
<div>
  <div
    class="banner"
    id="banner"
  >
    <div
      data-testid="signature"
      id="signature"
    >
      Made by Andrew Moy
    </div>
    <div
      data-testid="title"
      id="title"
    >
      Battleship
    </div>
  </div>
  <div
    class="battlefields"
  >
    <div
      data-testid="playerBoard"
      id="playerBoard"
    >
      <div
        class="battlefield"
      >
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-0-0"
          />
          <div
            class="box ship "
            data-testid="box-0-1"
          />
          <div
            class="box  "
            data-testid="box-0-2"
          />
          <div
            class="box  "
            data-testid="box-0-3"
          />
          <div
            class="box  "
            data-testid="box-0-4"
          />
          <div
            class="box  "
            data-testid="box-0-5"
          />
          <div
            class="box  "
            data-testid="box-0-6"
          />
          <div
            class="box  "
            data-testid="box-0-7"
          />
          <div
            class="box  "
            data-testid="box-0-8"
          />
          <div
            class="box  "
            data-testid="box-0-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-1-0"
          />
          <div
            class="box ship "
            data-testid="box-1-1"
          />
          <div
            class="box  "
            data-testid="box-1-2"
          />
          <div
            class="box  "
            data-testid="box-1-3"
          />
          <div
            class="box  "
            data-testid="box-1-4"
          />
          <div
            class="box  "
            data-testid="box-1-5"
          />
          <div
            class="box  "
            data-testid="box-1-6"
          />
          <div
            class="box  "
            data-testid="box-1-7"
          />
          <div
            class="box  "
            data-testid="box-1-8"
          />
          <div
            class="box  "
            data-testid="box-1-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-2-0"
          />
          <div
            class="box ship "
            data-testid="box-2-1"
          />
          <div
            class="box ship "
            data-testid="box-2-2"
          />
          <div
            class="box  "
            data-testid="box-2-3"
          />
          <div
            class="box  "
            data-testid="box-2-4"
          />
          <div
            class="box  "
            data-testid="box-2-5"
          />
          <div
            class="box  "
            data-testid="box-2-6"
          />
          <div
            class="box  "
            data-testid="box-2-7"
          />
          <div
            class="box  "
            data-testid="box-2-8"
          />
          <div
            class="box  "
            data-testid="box-2-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-3-0"
          />
          <div
            class="box ship "
            data-testid="box-3-1"
          />
          <div
            class="box ship "
            data-testid="box-3-2"
          />
          <div
            class="box ship "
            data-testid="box-3-3"
          />
          <div
            class="box  "
            data-testid="box-3-4"
          />
          <div
            class="box  "
            data-testid="box-3-5"
          />
          <div
            class="box  "
            data-testid="box-3-6"
          />
          <div
            class="box  "
            data-testid="box-3-7"
          />
          <div
            class="box  "
            data-testid="box-3-8"
          />
          <div
            class="box  "
            data-testid="box-3-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-4-0"
          />
          <div
            class="box ship "
            data-testid="box-4-1"
          />
          <div
            class="box ship "
            data-testid="box-4-2"
          />
          <div
            class="box ship "
            data-testid="box-4-3"
          />
          <div
            class="box ship "
            data-testid="box-4-4"
          />
          <div
            class="box  "
            data-testid="box-4-5"
          />
          <div
            class="box  "
            data-testid="box-4-6"
          />
          <div
            class="box  "
            data-testid="box-4-7"
          />
          <div
            class="box  "
            data-testid="box-4-8"
          />
          <div
            class="box  "
            data-testid="box-4-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-5-0"
          />
          <div
            class="box  "
            data-testid="box-5-1"
          />
          <div
            class="box  "
            data-testid="box-5-2"
          />
          <div
            class="box  "
            data-testid="box-5-3"
          />
          <div
            class="box  "
            data-testid="box-5-4"
          />
          <div
            class="box  "
            data-testid="box-5-5"
          />
          <div
            class="box  "
            data-testid="box-5-6"
          />
          <div
            class="box  "
            data-testid="box-5-7"
          />
          <div
            class="box  "
            data-testid="box-5-8"
          />
          <div
            class="box  "
            data-testid="box-5-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-6-0"
          />
          <div
            class="box  "
            data-testid="box-6-1"
          />
          <div
            class="box  "
            data-testid="box-6-2"
          />
          <div
            class="box  "
            data-testid="box-6-3"
          />
          <div
            class="box  "
            data-testid="box-6-4"
          />
          <div
            class="box  "
            data-testid="box-6-5"
          />
          <div
            class="box  "
            data-testid="box-6-6"
          />
          <div
            class="box  "
            data-testid="box-6-7"
          />
          <div
            class="box  "
            data-testid="box-6-8"
          />
          <div
            class="box  "
            data-testid="box-6-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-7-0"
          />
          <div
            class="box  "
            data-testid="box-7-1"
          />
          <div
            class="box  "
            data-testid="box-7-2"
          />
          <div
            class="box  "
            data-testid="box-7-3"
          />
          <div
            class="box  "
            data-testid="box-7-4"
          />
          <div
            class="box  "
            data-testid="box-7-5"
          />
          <div
            class="box  "
            data-testid="box-7-6"
          />
          <div
            class="box  "
            data-testid="box-7-7"
          />
          <div
            class="box  "
            data-testid="box-7-8"
          />
          <div
            class="box  "
            data-testid="box-7-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-8-0"
          />
          <div
            class="box  "
            data-testid="box-8-1"
          />
          <div
            class="box  "
            data-testid="box-8-2"
          />
          <div
            class="box  "
            data-testid="box-8-3"
          />
          <div
            class="box  "
            data-testid="box-8-4"
          />
          <div
            class="box  "
            data-testid="box-8-5"
          />
          <div
            class="box  "
            data-testid="box-8-6"
          />
          <div
            class="box  "
            data-testid="box-8-7"
          />
          <div
            class="box  "
            data-testid="box-8-8"
          />
          <div
            class="box  "
            data-testid="box-8-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-9-0"
          />
          <div
            class="box  "
            data-testid="box-9-1"
          />
          <div
            class="box  "
            data-testid="box-9-2"
          />
          <div
            class="box  "
            data-testid="box-9-3"
          />
          <div
            class="box  "
            data-testid="box-9-4"
          />
          <div
            class="box  "
            data-testid="box-9-5"
          />
          <div
            class="box  "
            data-testid="box-9-6"
          />
          <div
            class="box  "
            data-testid="box-9-7"
          />
          <div
            class="box  "
            data-testid="box-9-8"
          />
          <div
            class="box  "
            data-testid="box-9-9"
          />
        </div>
      </div>
      <div
        class="battlefield"
      >
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-0-0"
          />
          <div
            class="box "
            data-testid="box-0-1"
          />
          <div
            class="box "
            data-testid="box-0-2"
          />
          <div
            class="box "
            data-testid="box-0-3"
          />
          <div
            class="box "
            data-testid="box-0-4"
          />
          <div
            class="box "
            data-testid="box-0-5"
          />
          <div
            class="box "
            data-testid="box-0-6"
          />
          <div
            class="box "
            data-testid="box-0-7"
          />
          <div
            class="box "
            data-testid="box-0-8"
          />
          <div
            class="box "
            data-testid="box-0-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-1-0"
          />
          <div
            class="box "
            data-testid="box-1-1"
          />
          <div
            class="box "
            data-testid="box-1-2"
          />
          <div
            class="box "
            data-testid="box-1-3"
          />
          <div
            class="box "
            data-testid="box-1-4"
          />
          <div
            class="box "
            data-testid="box-1-5"
          />
          <div
            class="box "
            data-testid="box-1-6"
          />
          <div
            class="box "
            data-testid="box-1-7"
          />
          <div
            class="box "
            data-testid="box-1-8"
          />
          <div
            class="box "
            data-testid="box-1-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-2-0"
          />
          <div
            class="box "
            data-testid="box-2-1"
          />
          <div
            class="box "
            data-testid="box-2-2"
          />
          <div
            class="box "
            data-testid="box-2-3"
          />
          <div
            class="box "
            data-testid="box-2-4"
          />
          <div
            class="box "
            data-testid="box-2-5"
          />
          <div
            class="box "
            data-testid="box-2-6"
          />
          <div
            class="box "
            data-testid="box-2-7"
          />
          <div
            class="box "
            data-testid="box-2-8"
          />
          <div
            class="box "
            data-testid="box-2-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-3-0"
          />
          <div
            class="box "
            data-testid="box-3-1"
          />
          <div
            class="box "
            data-testid="box-3-2"
          />
          <div
            class="box "
            data-testid="box-3-3"
          />
          <div
            class="box "
            data-testid="box-3-4"
          />
          <div
            class="box "
            data-testid="box-3-5"
          />
          <div
            class="box "
            data-testid="box-3-6"
          />
          <div
            class="box "
            data-testid="box-3-7"
          />
          <div
            class="box "
            data-testid="box-3-8"
          />
          <div
            class="box "
            data-testid="box-3-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-4-0"
          />
          <div
            class="box "
            data-testid="box-4-1"
          />
          <div
            class="box "
            data-testid="box-4-2"
          />
          <div
            class="box "
            data-testid="box-4-3"
          />
          <div
            class="box "
            data-testid="box-4-4"
          />
          <div
            class="box "
            data-testid="box-4-5"
          />
          <div
            class="box "
            data-testid="box-4-6"
          />
          <div
            class="box "
            data-testid="box-4-7"
          />
          <div
            class="box "
            data-testid="box-4-8"
          />
          <div
            class="box "
            data-testid="box-4-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-5-0"
          />
          <div
            class="box "
            data-testid="box-5-1"
          />
          <div
            class="box "
            data-testid="box-5-2"
          />
          <div
            class="box "
            data-testid="box-5-3"
          />
          <div
            class="box "
            data-testid="box-5-4"
          />
          <div
            class="box "
            data-testid="box-5-5"
          />
          <div
            class="box "
            data-testid="box-5-6"
          />
          <div
            class="box "
            data-testid="box-5-7"
          />
          <div
            class="box "
            data-testid="box-5-8"
          />
          <div
            class="box "
            data-testid="box-5-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-6-0"
          />
          <div
            class="box "
            data-testid="box-6-1"
          />
          <div
            class="box "
            data-testid="box-6-2"
          />
          <div
            class="box "
            data-testid="box-6-3"
          />
          <div
            class="box "
            data-testid="box-6-4"
          />
          <div
            class="box "
            data-testid="box-6-5"
          />
          <div
            class="box "
            data-testid="box-6-6"
          />
          <div
            class="box "
            data-testid="box-6-7"
          />
          <div
            class="box "
            data-testid="box-6-8"
          />
          <div
            class="box "
            data-testid="box-6-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-7-0"
          />
          <div
            class="box "
            data-testid="box-7-1"
          />
          <div
            class="box "
            data-testid="box-7-2"
          />
          <div
            class="box "
            data-testid="box-7-3"
          />
          <div
            class="box "
            data-testid="box-7-4"
          />
          <div
            class="box "
            data-testid="box-7-5"
          />
          <div
            class="box "
            data-testid="box-7-6"
          />
          <div
            class="box "
            data-testid="box-7-7"
          />
          <div
            class="box "
            data-testid="box-7-8"
          />
          <div
            class="box "
            data-testid="box-7-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-8-0"
          />
          <div
            class="box "
            data-testid="box-8-1"
          />
          <div
            class="box "
            data-testid="box-8-2"
          />
          <div
            class="box "
            data-testid="box-8-3"
          />
          <div
            class="box "
            data-testid="box-8-4"
          />
          <div
            class="box "
            data-testid="box-8-5"
          />
          <div
            class="box "
            data-testid="box-8-6"
          />
          <div
            class="box "
            data-testid="box-8-7"
          />
          <div
            class="box "
            data-testid="box-8-8"
          />
          <div
            class="box "
            data-testid="box-8-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-9-0"
          />
          <div
            class="box "
            data-testid="box-9-1"
          />
          <div
            class="box "
            data-testid="box-9-2"
          />
          <div
            class="box "
            data-testid="box-9-3"
          />
          <div
            class="box "
            data-testid="box-9-4"
          />
          <div
            class="box "
            data-testid="box-9-5"
          />
          <div
            class="box "
            data-testid="box-9-6"
          />
          <div
            class="box "
            data-testid="box-9-7"
          />
          <div
            class="box "
            data-testid="box-9-8"
          />
          <div
            class="box "
            data-testid="box-9-9"
          />
        </div>
      </div>
    </div>
    <div
      data-testid="computerBoard"
      id="computerBoard"
    >
      <div
        class="battlefield"
      >
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-0-0"
          />
          <div
            class="box ship "
            data-testid="box-0-1"
          />
          <div
            class="box  "
            data-testid="box-0-2"
          />
          <div
            class="box  "
            data-testid="box-0-3"
          />
          <div
            class="box  "
            data-testid="box-0-4"
          />
          <div
            class="box  "
            data-testid="box-0-5"
          />
          <div
            class="box  "
            data-testid="box-0-6"
          />
          <div
            class="box  "
            data-testid="box-0-7"
          />
          <div
            class="box  "
            data-testid="box-0-8"
          />
          <div
            class="box  "
            data-testid="box-0-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-1-0"
          />
          <div
            class="box ship "
            data-testid="box-1-1"
          />
          <div
            class="box  "
            data-testid="box-1-2"
          />
          <div
            class="box  "
            data-testid="box-1-3"
          />
          <div
            class="box  "
            data-testid="box-1-4"
          />
          <div
            class="box  "
            data-testid="box-1-5"
          />
          <div
            class="box  "
            data-testid="box-1-6"
          />
          <div
            class="box  "
            data-testid="box-1-7"
          />
          <div
            class="box  "
            data-testid="box-1-8"
          />
          <div
            class="box  "
            data-testid="box-1-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-2-0"
          />
          <div
            class="box ship "
            data-testid="box-2-1"
          />
          <div
            class="box ship "
            data-testid="box-2-2"
          />
          <div
            class="box  "
            data-testid="box-2-3"
          />
          <div
            class="box  "
            data-testid="box-2-4"
          />
          <div
            class="box  "
            data-testid="box-2-5"
          />
          <div
            class="box  "
            data-testid="box-2-6"
          />
          <div
            class="box  "
            data-testid="box-2-7"
          />
          <div
            class="box  "
            data-testid="box-2-8"
          />
          <div
            class="box  "
            data-testid="box-2-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-3-0"
          />
          <div
            class="box ship "
            data-testid="box-3-1"
          />
          <div
            class="box ship "
            data-testid="box-3-2"
          />
          <div
            class="box ship "
            data-testid="box-3-3"
          />
          <div
            class="box  "
            data-testid="box-3-4"
          />
          <div
            class="box  "
            data-testid="box-3-5"
          />
          <div
            class="box  "
            data-testid="box-3-6"
          />
          <div
            class="box  "
            data-testid="box-3-7"
          />
          <div
            class="box  "
            data-testid="box-3-8"
          />
          <div
            class="box  "
            data-testid="box-3-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box ship "
            data-testid="box-4-0"
          />
          <div
            class="box ship "
            data-testid="box-4-1"
          />
          <div
            class="box ship "
            data-testid="box-4-2"
          />
          <div
            class="box ship "
            data-testid="box-4-3"
          />
          <div
            class="box ship "
            data-testid="box-4-4"
          />
          <div
            class="box  "
            data-testid="box-4-5"
          />
          <div
            class="box  "
            data-testid="box-4-6"
          />
          <div
            class="box  "
            data-testid="box-4-7"
          />
          <div
            class="box  "
            data-testid="box-4-8"
          />
          <div
            class="box  "
            data-testid="box-4-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-5-0"
          />
          <div
            class="box  "
            data-testid="box-5-1"
          />
          <div
            class="box  "
            data-testid="box-5-2"
          />
          <div
            class="box  "
            data-testid="box-5-3"
          />
          <div
            class="box  "
            data-testid="box-5-4"
          />
          <div
            class="box  "
            data-testid="box-5-5"
          />
          <div
            class="box  "
            data-testid="box-5-6"
          />
          <div
            class="box  "
            data-testid="box-5-7"
          />
          <div
            class="box  "
            data-testid="box-5-8"
          />
          <div
            class="box  "
            data-testid="box-5-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-6-0"
          />
          <div
            class="box  "
            data-testid="box-6-1"
          />
          <div
            class="box  "
            data-testid="box-6-2"
          />
          <div
            class="box  "
            data-testid="box-6-3"
          />
          <div
            class="box  "
            data-testid="box-6-4"
          />
          <div
            class="box  "
            data-testid="box-6-5"
          />
          <div
            class="box  "
            data-testid="box-6-6"
          />
          <div
            class="box  "
            data-testid="box-6-7"
          />
          <div
            class="box  "
            data-testid="box-6-8"
          />
          <div
            class="box  "
            data-testid="box-6-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-7-0"
          />
          <div
            class="box  "
            data-testid="box-7-1"
          />
          <div
            class="box  "
            data-testid="box-7-2"
          />
          <div
            class="box  "
            data-testid="box-7-3"
          />
          <div
            class="box  "
            data-testid="box-7-4"
          />
          <div
            class="box  "
            data-testid="box-7-5"
          />
          <div
            class="box  "
            data-testid="box-7-6"
          />
          <div
            class="box  "
            data-testid="box-7-7"
          />
          <div
            class="box  "
            data-testid="box-7-8"
          />
          <div
            class="box  "
            data-testid="box-7-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-8-0"
          />
          <div
            class="box  "
            data-testid="box-8-1"
          />
          <div
            class="box  "
            data-testid="box-8-2"
          />
          <div
            class="box  "
            data-testid="box-8-3"
          />
          <div
            class="box  "
            data-testid="box-8-4"
          />
          <div
            class="box  "
            data-testid="box-8-5"
          />
          <div
            class="box  "
            data-testid="box-8-6"
          />
          <div
            class="box  "
            data-testid="box-8-7"
          />
          <div
            class="box  "
            data-testid="box-8-8"
          />
          <div
            class="box  "
            data-testid="box-8-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box  "
            data-testid="box-9-0"
          />
          <div
            class="box  "
            data-testid="box-9-1"
          />
          <div
            class="box  "
            data-testid="box-9-2"
          />
          <div
            class="box  "
            data-testid="box-9-3"
          />
          <div
            class="box  "
            data-testid="box-9-4"
          />
          <div
            class="box  "
            data-testid="box-9-5"
          />
          <div
            class="box  "
            data-testid="box-9-6"
          />
          <div
            class="box  "
            data-testid="box-9-7"
          />
          <div
            class="box  "
            data-testid="box-9-8"
          />
          <div
            class="box  "
            data-testid="box-9-9"
          />
        </div>
      </div>
      <div
        class="battlefield"
      >
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-0-0"
          />
          <div
            class="box "
            data-testid="box-0-1"
          />
          <div
            class="box "
            data-testid="box-0-2"
          />
          <div
            class="box "
            data-testid="box-0-3"
          />
          <div
            class="box "
            data-testid="box-0-4"
          />
          <div
            class="box "
            data-testid="box-0-5"
          />
          <div
            class="box "
            data-testid="box-0-6"
          />
          <div
            class="box "
            data-testid="box-0-7"
          />
          <div
            class="box "
            data-testid="box-0-8"
          />
          <div
            class="box "
            data-testid="box-0-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-1-0"
          />
          <div
            class="box "
            data-testid="box-1-1"
          />
          <div
            class="box "
            data-testid="box-1-2"
          />
          <div
            class="box "
            data-testid="box-1-3"
          />
          <div
            class="box "
            data-testid="box-1-4"
          />
          <div
            class="box "
            data-testid="box-1-5"
          />
          <div
            class="box "
            data-testid="box-1-6"
          />
          <div
            class="box "
            data-testid="box-1-7"
          />
          <div
            class="box "
            data-testid="box-1-8"
          />
          <div
            class="box "
            data-testid="box-1-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-2-0"
          />
          <div
            class="box "
            data-testid="box-2-1"
          />
          <div
            class="box "
            data-testid="box-2-2"
          />
          <div
            class="box "
            data-testid="box-2-3"
          />
          <div
            class="box "
            data-testid="box-2-4"
          />
          <div
            class="box "
            data-testid="box-2-5"
          />
          <div
            class="box "
            data-testid="box-2-6"
          />
          <div
            class="box "
            data-testid="box-2-7"
          />
          <div
            class="box "
            data-testid="box-2-8"
          />
          <div
            class="box "
            data-testid="box-2-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-3-0"
          />
          <div
            class="box "
            data-testid="box-3-1"
          />
          <div
            class="box "
            data-testid="box-3-2"
          />
          <div
            class="box "
            data-testid="box-3-3"
          />
          <div
            class="box "
            data-testid="box-3-4"
          />
          <div
            class="box "
            data-testid="box-3-5"
          />
          <div
            class="box "
            data-testid="box-3-6"
          />
          <div
            class="box "
            data-testid="box-3-7"
          />
          <div
            class="box "
            data-testid="box-3-8"
          />
          <div
            class="box "
            data-testid="box-3-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-4-0"
          />
          <div
            class="box "
            data-testid="box-4-1"
          />
          <div
            class="box "
            data-testid="box-4-2"
          />
          <div
            class="box "
            data-testid="box-4-3"
          />
          <div
            class="box "
            data-testid="box-4-4"
          />
          <div
            class="box "
            data-testid="box-4-5"
          />
          <div
            class="box "
            data-testid="box-4-6"
          />
          <div
            class="box "
            data-testid="box-4-7"
          />
          <div
            class="box "
            data-testid="box-4-8"
          />
          <div
            class="box "
            data-testid="box-4-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-5-0"
          />
          <div
            class="box "
            data-testid="box-5-1"
          />
          <div
            class="box "
            data-testid="box-5-2"
          />
          <div
            class="box "
            data-testid="box-5-3"
          />
          <div
            class="box "
            data-testid="box-5-4"
          />
          <div
            class="box "
            data-testid="box-5-5"
          />
          <div
            class="box "
            data-testid="box-5-6"
          />
          <div
            class="box "
            data-testid="box-5-7"
          />
          <div
            class="box "
            data-testid="box-5-8"
          />
          <div
            class="box "
            data-testid="box-5-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-6-0"
          />
          <div
            class="box "
            data-testid="box-6-1"
          />
          <div
            class="box "
            data-testid="box-6-2"
          />
          <div
            class="box "
            data-testid="box-6-3"
          />
          <div
            class="box "
            data-testid="box-6-4"
          />
          <div
            class="box "
            data-testid="box-6-5"
          />
          <div
            class="box "
            data-testid="box-6-6"
          />
          <div
            class="box "
            data-testid="box-6-7"
          />
          <div
            class="box "
            data-testid="box-6-8"
          />
          <div
            class="box "
            data-testid="box-6-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-7-0"
          />
          <div
            class="box "
            data-testid="box-7-1"
          />
          <div
            class="box "
            data-testid="box-7-2"
          />
          <div
            class="box "
            data-testid="box-7-3"
          />
          <div
            class="box "
            data-testid="box-7-4"
          />
          <div
            class="box "
            data-testid="box-7-5"
          />
          <div
            class="box "
            data-testid="box-7-6"
          />
          <div
            class="box "
            data-testid="box-7-7"
          />
          <div
            class="box "
            data-testid="box-7-8"
          />
          <div
            class="box "
            data-testid="box-7-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-8-0"
          />
          <div
            class="box "
            data-testid="box-8-1"
          />
          <div
            class="box "
            data-testid="box-8-2"
          />
          <div
            class="box "
            data-testid="box-8-3"
          />
          <div
            class="box "
            data-testid="box-8-4"
          />
          <div
            class="box "
            data-testid="box-8-5"
          />
          <div
            class="box "
            data-testid="box-8-6"
          />
          <div
            class="box "
            data-testid="box-8-7"
          />
          <div
            class="box "
            data-testid="box-8-8"
          />
          <div
            class="box "
            data-testid="box-8-9"
          />
        </div>
        <div
          class="battlefieldCol"
        >
          <div
            class="box "
            data-testid="box-9-0"
          />
          <div
            class="box "
            data-testid="box-9-1"
          />
          <div
            class="box "
            data-testid="box-9-2"
          />
          <div
            class="box "
            data-testid="box-9-3"
          />
          <div
            class="box "
            data-testid="box-9-4"
          />
          <div
            class="box "
            data-testid="box-9-5"
          />
          <div
            class="box "
            data-testid="box-9-6"
          />
          <div
            class="box "
            data-testid="box-9-7"
          />
          <div
            class="box "
            data-testid="box-9-8"
          />
          <div
            class="box "
            data-testid="box-9-9"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
